### PR TITLE
Added function to log out on severe justice messages

### PR DESCRIPTION
**Pull request by adamgoerdt**

### CODE REVIEW

**Comment by jandersson** [comment]
Any chance there could be a setting for the threshold? It looks like its 4, but as a necromancer I'd like it to be 1 since it puts me in PvP open.

**Comment by adamgoerdt** [comment]
Easy enough if you can help me think of a name for the setting

**Comment by jandersson** [comment]
`afk_justice_threshold`?

**Comment by jandersson** [comment]
Or lump all afk settings into a dictionary/hashmap:

```
afk:
    justice_threshold: 4
```

**Comment by adamgoerdt** [comment]
I left it as afk_justice_threshold instead of adding the afk hierarchy. There are already settings (health_threshold and depart_on_death) in afk which are in wide use.  They can't really be migrated to a hierarchy without breaking people's existing setups

File: afk.lic [hunk 1]
```diff
@@ -22,11 +22,21 @@ depart_on_death = settings.depart_on_death
...
   pause 0.05 unless line
 
   fput(%w(tdp time age).sample) if line =~ /you have been idle too long/i
+  
+  justice_message_count += 1 if line =~ /You hear the cries echo around you as everyone in the vicinity immediately gives/
+  justice_message_count += 1 if line =~ /authorities will try to bring you in for endangering the public/
+  
+  if justice_message_count > settings.afk_justice_threshold
+    echo "Looks like you've run into trouble with the law.  Exiting"
+    fput('exit')
```
Comment (rpherbig) [inline afk.lic]: Should this call `exit_game` instead?

File: afk.lic [hunk 2]
```diff
@@ -22,11 +22,21 @@ depart_on_death = settings.depart_on_death
...
+  end
 
   if Room.current.id == 9610
     DRC.fix_standing
```
Comment (rpherbig) [inline afk.lic]: Should this (and line 43) be `move('out')` instead?

File: afk.lic [hunk 3]
```diff
@@ -22,11 +22,21 @@ depart_on_death = settings.depart_on_death
...
 loop do
   line = script.gets?
   pause 0.05 unless line
 
   fput(%w(tdp time age).sample) if line =~ /you have been idle too long/i
+  
+  justice_message_count += 1 if line =~ /You hear the cries echo around you as everyone in the vicinity immediately gives/
+  justice_message_count += 1 if line =~ /authorities will try to bring you in for endangering the public/
+  
+  if justice_message_count > settings.afk_justice_threshold
```
Comment (rpherbig) [inline afk.lic]: What do you think about defining `justice_threshold = settings.afk_justice_threshold` on line 22?

**Pull request merged**